bzr.go

```diff
@@ -146,6 +147,40 @@ func (s *BzrRepo) Tags() ([]string, error) {
 	return tags, nil
 }
 
+// CurrentVersionsWithRevs returns a list of available branches and tags, and
+// their underlying revision identifiers. The list is guaranteed to be current.
+//
+// As the model we use for bzr interaction (and go get generally does as well)
+// involves interacting only with a bzr repository's default branch, we return
+// only tags here.
+func (s *BzrRepo) CurrentVersionsWithRevs() (versions []VersionInfo, localSynced bool, err error) {
+	var out []byte
+	// Update the local first
+	_, err = s.runFromDir("bzr", "pull")
+	if err != nil {
+		return
+	}
+	localSynced = true
```

Comment:
I'm confused. What is the purpose of `localSynced` as it relates to the calling code?

One of my goals is to keep the API simple and hide the complexity. There shouldn't be any WTF moments to someone using the library. Right now I don't understand how `localSynced` relates to the overall developer experience. Can you provide some insight?